docs/reference/contracts/apis/managed.md

```diff
@@ -0,0 +1,294 @@
+
+### `isAddressOnAllowlist`
+
+```solidity
+isAddressOnAllowlist(address member) 
+returns (bool)
+```
+
+Returns whether a given address, `member`, is on the allowlist. Checks the allowlist regardless of whether the allowlist features is enabled. Implemented in `ManagedPoolSettings`.
+
+### `addAllowedAddress`
+
+```solidity
+addAllowedAddress(address member)
+
+emits AllowlistAddressAdded(address indexed member)
+```
+
+Adds an address, `member`, to the LP allowlist. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+### `removeAllowedAddress`
+
+```solidity
+removeAllowedAddress(address member)
+
+emits AllowlistAddressRemoved(address indexed member)
+```
+
+Removes an address, `member`, from the LP allowlist. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+### `setMustAllowlistLPs`
+
+```solidity
+setMustAllowlistLPs(bool mustAllowlistLPs)
+
+emit MustAllowlistLPsSet(bool mustAllowlistLPs)
+```
+
+Enables or disables the LP allowlist. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+## AUM Fees
+
+### `getManagementAumFeeParams`
+
+```solidity
+getManagementAumFeeParams()
+returns (uint256 aumFeePercentage, uint256 lastCollectionTimestamp)
+```
+
+Returns the AUM fee percentage as an 18-decimal fixed point number, and the timestamp of the last collection of AUM fees. Implemented in `ManagedPoolSettings`.
+
+###  `setManagementAumFeePercentage`
+
+```solidity
+setManagementAumFeePercentage(uint256 managementAumFeePercentage)
+returns (uint256)
+
+emits ManagementAumFeePercentageChanged(uint256 managementAumFeePercentage)
+```
+
+Sets the yearly percentage AUM fee, which is payable to the pool `owner`. The amount of BPT minted to the manager before the update is returned. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+###  `collectAumManagementFees`
+
+```solidity
+collectAumManagementFees()
+
+emits ManagementAumFeeCollected(uint256 bptAmount)
+```
+
+Collects any accrued AUM fees and sends them to the Managed pool `owner`. It will also be called automatically whenever supply changes occur (e.g., joins/exits, add/remove token) and before any fee percentage changes. This is not a permissioned function and can be called by anyone. Implemented in `ManagedPoolSettings`.
+
+## Circuit Breakers
+
+### `getCircuitBreakerState`
+
+```solidity
+getCircuitBreakerState(IERC20 token)
+returns (uint256 bptPrice,
+         uint256 referenceWeight,
+         uint256 lowerBound,
+         uint256 upperBound,
+         uint256 lowerBptPriceBound,
+         uint256 upperBptPriceBound)
+```
+
+Returns the full circuit breaker state for a given `token`. Implemented in `ManagedPoolSettings`.
```

Comment:
add notes that:
* `bptPrice` is the price of the BPT denominated in `token`
* `referenceWeight` is the weight of `token` when the cb was set

I personally think the other terms should be self-explanatory, but if you think it looks weird to only explain a few values, feel free to explain them all